ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -82,6 +83,9 @@ object SandboxServer {
 
 class SandboxServer(actorSystemName: String, config: => SandboxConfig) extends AutoCloseable {
 
+  // Name of this participant, ultimately pass this info in command-line
+  val participantId: ParticipantId = Ref.LedgerString.assertFromString("in-memory-participant")
```

Comment:
Please use `sandbox` as the participant id.

Reply:
Done